File: week1/day4/demo/grades.rb
#Write a piece of code that asks the user for their grade on an exam then prints either A, B, C, D or F

puts 'What was Your Score (0 - 100) on the Exam? '
number = gets.to_i

output = if number > 101
  "A++ must have been Bonus Points, Great Work!"
elsif number > 90
  "Congratulations, you got an A!!!"
elsif number > 75
  "Good Job, that's a B"
elsif number > 60
  "Alright, its a C"
elsif number > 50
  "That's a D... study more next time."
else
  "F ... you didn't Study enough!"
end

puts output

File: week4/day2/labs/grades/grades.js
const functions = {
  grading: function (score) {
    // let grade = parseInt(score);
      if (score >= 95) {
        return `Congratulations, you got an A+!!`;
      } else if (score >= 90) {
        return `Congratualtions, you got an A!`;
      } else if (score >= 80) {
        return `You did Alright... That's a B Grade`;
      } else if (score >= 70) {
        return `Not Bad, you got a C+`;
      } else if (score >= 60) {
        return `At least that you got a C`;
      } else if (score >= 50) {
        return `Well at least D is a pass`;
      } else {
        return `That's an F... study harder Fool`;
      }
    }
  };